app/models/shipwire/orders_decorator.rb

```diff
@@ -0,0 +1,5 @@
+Shipwire::Orders.class_eval do
+  def find(id, params = {})
```

Comment:
As discussed, why don't you add a comment explaining the problem with the Shipwire gem?